app/views/comments/destroy.js.erb

```diff
@@ -2,4 +2,4 @@
 $("#comment<%= @comment.id.to_s %>").fadeOut(300, function() { $(this).remove(); });
 
 /* Update comment count */
-$("#comment_count").html("(<%= @comment.listing.comments_count - 1 %>)");
+$("#comment_count").html("(<%= @comment.listing.comments_count %>)");
```

Comment:
seems that rails 5 has fixed some data reloading, so that the -1 is not needed anymore